nipype/interfaces/fsl/epi.py

```diff
@@ -140,6 +140,13 @@ class TOPUPInputSpec(FSLCommandInputSpec):
     out_field = File(argstr='--fout=%s', hash_files=False,
                      name_source=['in_file'], name_template='%s_field',
                      desc='name of image file with field (Hz)')
+    out_warp_prefix = traits.Str("warpfield", argstr='--dfout=%s', hash_files=False,
+                                 desc='prefix for the warpfield images (in mm)',
+                                 usedefault=True)
+    out_jac_prefix = traits.Str("jac", argstr='--jacout=%s',
+                                 hash_files=False,
+                                 desc='prefix for the warpfield images',
+                                 usedefault=True)
```

Comment:
Does making these required outputs have the potential to produce large amounts of data?